tests/ffi/tests.cc

```diff
@@ -401,6 +401,12 @@ extern "C" const char *cxx_run_test() noexcept {
       std::unique_ptr<std::string>(new std::string("2020")));
   r_take_enum(Enum::AVal);
 
+  // Caller owns data handed into rust APIs
```

Comment:
Test for that did not really fit into the "take" or "return" tests. 

I initially had a c_take...() test that would create a std::vector and then call into the r_access_...() test function, but I found that overly complex as the c_take...() test did not add anything. Hence, directly testing the API in r_access..() here.